examples/next/pages/ui/components/account-settings/change-password/index.page.tsx

```diff
@@ -10,38 +10,38 @@ import {
   Flex,
   Heading,
   AccountSettings,
+  withAuthenticator,
```

Comment:
unused import alert ;)

Reply:
🤦 , thank you.

Reply:
Fixed in [9f4223c](https://github.com/aws-amplify/amplify-ui/pull/3425/commits/9f4223c3a91f5ee14d1e2e84c1fa61793911bbb8)

Reply:
I'm also curious why this isn't caught by eslint?